server.js

```diff
@@ -237,7 +238,7 @@ module.exports = function(options) {
       }
     }, {
       method: 'GET',
-      path: '/api/paypal-redirect/{frequency}/{locale}/',
+      path: '/api/paypal-redirect/{frequency}/{locale}/{appName}/',
```

Comment:
For some reason `{appName?}` didn't work as optional, so I ensure we pass in "thunderbird" or "mozillafoundation" so that something is always here and not optional. I think in the long run this is better though as it means if we ever need a fourth param, we don't need to trip over an optional third param.